connectors-home/modules/ROOT/pages/introduction/intro-config-oauth2-cloudhub.adoc

```diff
@@ -0,0 +1,39 @@
+= Configuring OAuth 2.0 for Connectors on CloudHub
+
+You can deploy your OAuth 2.0 enabled connectors to a cloud service. Learn the required configuration for the OAuth 2.0 setup and how to deploy your connector to CloudHub.
+
+== How to Configure OAuth 2.0 for a Connector on CloudHub
+
+Configuring OAuth 2.0 for a connector on CloudHub is the same as configuring OAuth 2.0 for a connector locally, except with CloudHub you are behind an extra server. You must keep in mind the following when configuring OAuth 2.0 for a connector on CloudHub:
```

Comment:
```suggestion
Configuring OAuth 2.0 for a connector on CloudHub is the same as configuring OAuth 2.0 for a connector locally, except with CloudHub, you are behind an extra server. Consider the following when configuring OAuth 2.0 for a connector on CloudHub:
```

Reply:
"You" are not behind an extra server, the Mule implementation is. I am not sure how to word this - the developer can probably help.